app/assets/javascripts/notification.js.coffee

```diff
@@ -0,0 +1,11 @@
+$(document).on 'click', '[data-behavior~=update-is_read]', (_this) ->
+  $.ajax
+    type: 'PUT'
+    url: '/notifications/' + @id
+    data:
+      is_read: @checked
+      id: @id
+  if @checked
```

Comment:
I think it better call it in terms of meaning and not actual control representation. Something like `read` or `is_read`.
Or is it a JQuery name that we cannot change?